src/mappings/utils.ts

```diff
@@ -37,6 +38,14 @@ export function fetchTokenSymbol(tokenAddress: Address): string {
     return 'AAVE'
   }
 
+  if (tokenAddress.toHexString() == KNC_ADDRESS) {
```

Comment:
Why `==`?

Reply:
Because of https://github.com/AssemblyScript/assemblyscript/issues/621#issuecomment-496990742